test/integration/regression_go_bin_scanner_arch_test.go

```diff
@@ -9,9 +9,9 @@ import (
 
 func TestRegressionGoArchDiscovery(t *testing.T) {
 	const (
-		expectedELFPkg   = 3
-		expectedWINPkg   = 3
-		expectedMACOSPkg = 3
+		expectedELFPkg   = 4
```

Comment:
what changed that caused these numbers to be bumped?

Reply:
we now add the main package

Reply:
excellent 👍